apps/web-giddh/src/app/settings/permissions/form/form.component.html

```diff
@@ -1,143 +1,219 @@
-        <!-- <select name="roleUniqueName" formControlName="roleUniqueName" class="form-control ">
-        <option [value]="role.uniqueName" *ngFor="let role of allRoles">{{role.name}}</option>
-    </select> -->
-        <sh-select placeholder="Select role" formControlName="roleUniqueName" [options]="allRoles"></sh-select>
+    <div class="clearfix">
+        <p class="bold font-14 mb-1 d-inline-block middle mr-1">Specific Permission</p>
+        <div class="relative d-inline-block transactions-limit-box">
+            <a href="javascript:void(0)" class="text-gray text-gray pr-1 d-inline-block"><i class="icon-info"></i></a>
+            <div class="transactions-popup">
+                <p class="bold mb-1">Bank Account</p>
+                <p class="font-12 text-gray mb-1"> Aenean non accumsan ante. Duis et risus accumsan sem tempus porta nec sit amet est. Sed ut euismod quam. Suspendisse potenti. Aliquam fringilla orci tincidunt, ullamcorper erat in, malesuada metus.</p>
+                <a href="javascript:void(0)" class="btn-link bold font-14">More Details</a>
+            </div>
+        </div>
     </div>
-
-    <!-- time span -->
-    <div class="form-group" [exclude]="'.time-span-dropdown'" [attachOutsideOnClick]="true" (clickOutside)="methodForToggleSection('timeSpanSection')">
-        <label *ngIf="isLblShow">Time Span</label>
-        <div class="btn-group" [ngClass]="{'open': showTimeSpan, 'btn-group-liq': !isHorizntl}">
-            <button (click)="showTimeSpan = !showTimeSpan" type="button" class="form-control text-left ">
-                {{selectedTimeSpan}}
-                <span class="pull-right"><span class="fa fa-caret-down"></span></span>
-            </button>
-            <section class="dropdown-menu custom_dropdown mrB1 time-span-dropdown" [ngClass]="{'open': showTimeSpan}">
-
-                <div class="clearfix pd1 option_box">
-
-                    <span class="pd0 mrR3" (click)="isOpenedInModal ? handleTimeSpanChange('daterange'): ''">
+    <div class="d-flex row mb-15 permission-row">
+        <!-- time span -->
+        <div class="form-group col-xs-12" [exclude]="'.time-span-dropdown'" [attachOutsideOnClick]="true" (clickOutside)="methodForToggleSection('timeSpanSection')">
+            <label *ngIf="isLblShow">Time Span</label>
+            <div class="btn-group" [ngClass]="{'open': showTimeSpan, 'btn-group-liq': !isHorizntl}">
+                <button (click)="showTimeSpan = !showTimeSpan" type="button" class="form-control text-left">
+                    {{selectedTimeSpan}}
+                    <span class="pull-right"><span class="fa fa-caret-down"></span></span>
+                </button>
+                <ul class="dropdown-menu mb-1 time-span-dropdown" [ngClass]="{'open': showTimeSpan}">
+                    <li><a class="dropdown-item" href="javascript:void(0)" (click)="isOpenedInModal ? handleTimeSpanChange('daterange'): ''"><div class="item">Date Range</div></a></li>
```

Comment:
Add to JSON

Reply:
Already add JSON for **Date Range** content